<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>仿宋</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>N1 Auto manage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>424</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="BTN_ADD_RP">
       <property name="text">
        <string>添加&amp;RP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="BTN_MODIFY">
       <property name="text">
        <string>修改&amp;M</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BTN_ADD_AP">
       <property name="text">
        <string>添加&amp;AP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="BTN_ADD_Sensor">
       <property name="text">
        <string>添加&amp;Sensor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="BTN_DEL">
       <property name="text">
        <string>删除&amp;D</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>设备ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上行通道</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>下行通道</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="saveconfig">
       <property name="text">
        <string>保存配置&amp;V</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPushButton" name="openconfig">
       <property name="text">
        <string>加载配置&amp;O</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>设备配置表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>20</width>
      <height>731</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_adjust_sensor_list">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>160</y>
      <width>71</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Synpacket_number">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>680</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="CLBtn_openDeviceTable">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>显示设备</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>351</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>313</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="BTN_Connect">
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ip"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="BTN_GetAPparam">
          <property name="styleSheet">
           <string notr="true">alternate-background-color: rgb(170, 255, 127);
border-color: rgb(255, 85, 0);
background-color: rgb(255, 255, 0);</string>
          </property>
          <property name="text">
           <string>获取AP参数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>传输间隔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="LED_SendInterval">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>通道</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="LED_Channel">
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="text_adjust_level">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_adjust_timeout">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>400</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_adjust_rplist">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_adjust_level_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_adjust_level_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_count_down">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>160</y>
      <width>64</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>136</width>
      <height>56</height>
     </rect>
    </property>
    <property name="title">
     <string>自动配置时间槽</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="BTN_Adjust_slot_begin">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BTN_Adjust_slot_stop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>90</y>
      <width>292</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>设置通道</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="BTN_Adjust">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="BTN_Adjust_stop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="BTN_Adjust_pause">
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="BTN_Adjust_to_back">
          <property name="text">
           <string>回调</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>782</x>
      <y>90</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>倒计时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>RP列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>检测器列表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>168</width>
      <height>56</height>
     </rect>
    </property>
    <property name="title">
     <string>RP and Sensor</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="BTN_band">
       <property name="text">
        <string>绑定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BTN_band_false">
       <property name="text">
        <string>取消绑定</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>610</y>
      <width>168</width>
      <height>56</height>
     </rect>
    </property>
    <property name="title">
     <string>AP</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QRadioButton" name="BTN_band_ap">
       <property name="text">
        <string>绑定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BTN_band_false_AP">
       <property name="text">
        <string>取消绑定</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="BTN_save_ap_pram">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>680</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保存AP参数</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>line</zorder>
   <zorder>text_adjust_sensor_list</zorder>
   <zorder>lcdNumber_Synpacket_number</zorder>
   <zorder>CLBtn_openDeviceTable</zorder>
   <zorder>groupBox</zorder>
   <zorder>text_adjust_level</zorder>
   <zorder>text_adjust_timeout</zorder>
   <zorder>text_adjust_rplist</zorder>
   <zorder>text_adjust_level_2</zorder>
   <zorder>text_adjust_level_3</zorder>
   <zorder>lcdNumber_count_down</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>BTN_save_ap_pram</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_btn_add_rp_clicked()</slot>
 </slots>
</ui>
